### ISSUE TITLE
Workspace.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by nelson6e65** [jackytsu/atom-text-transformer#4]
Use ::getActivePaneItem() instead of the ::activePaneItem property

```
Workspace.Object.defineProperty.get (/usr/local/share/atom/resources/app.asar/src/workspace.js:915:14)
Object.lower (/home/nmartell/.atom/packages/atom-text-transformer/lib/atom-text-transformer.coffee:39:28)
atom-text-editor.<anonymous> (/home/nmartell/.atom/packages/atom-text-transformer/lib/atom-text-transformer.coffee:27:77)
CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/local/share/atom/resources/app.asar/src/command-registry.js:238:29)
handleCommandEvent (/usr/local/share/atom/resources/app.asar/src/command-registry.js:3:61)
CommandRegistry.dispatch (/usr/local/share/atom/resources/app.asar/src/command-registry.js:153:19)
```

**Comment by jackytsu** [comment]
fixed in v0.6.0

**Comment by nelson6e65** [comment]
:thumbsup:
